Makefile

```diff
@@ -11,6 +11,7 @@ COMPONENTS=cloud \
 
 .EXPORT_ALL_VARIABLES:
 OUT_DIR ?= _output
+GIT_TAG="1.2"
```

Comment:
does the GIT_TAG is necessary?  most of the cases for me to build those packages shall be based on master instead of specific version.

Reply:
The question is, when you want to build a new package (and release it) on every commit in the master or on every release. I think we should build the packages only on every release.

Reply:
@chendave @subpathdev  I believe the concept would be to release the Debian packages with every major release / git tag (1.2, 1.3 etc). So, maybe we can create the code which checks if a new git tag has been made and then build it. However, again, I don't have much experience in Makefile. I would appreciate if someone could fork and push the changes.